### ISSUE TITLE
listing of installed locales

### ISSUE THREAD

**Issue by kellyselden** [jamesarosen/ember-i18n#247]
I'm porting my projects from ember-cli-i18n. I'm about to port over my addon [ember-cli-i18n-locales](https://github.com/kellyselden/ember-cli-i18n-locales) which exposes a util function that returns the locale names installed for use in a locale chooser component. Any interest in a PR, or would you prefer to keep it out in an addon?

**Comment by jamesarosen** [comment]
I don't think a locale-chooser component is right, but adding a single public CP to `service:i18n` that lists the locales in the registry seems useful.

**Comment by kellyselden** [comment]
Right, the locale chooser component would be implemented in an app, not this addon.

**Comment by jamesarosen** [comment]
If you wanted, you could also emit a warning when changing `locale`:
```js
locale: Ember.computed({
  set(newLocale) {
    Ember.warn('No translations registered for ' + newLocale, function() {
      return this.get('availableLocales').some(function(loc) {
        return loc.indexOf(newLocale) === 0;
      });
    });

    return newLocale;
  }
})
```

**Comment by kellyselden** [comment]
fixed in #250
